dashboard/config/libraries/NativeSpriteLab.interpreted.js

```diff
@@ -167,36 +170,106 @@ function setupSim(
     };
   }
 
+  // We don't allow certain list functions in the interpreter apparently, so writing my own.
```

Comment:
Personal opinion - if you reference yourself in a comment, please sign the comment!